### PR TITLE
adding rspec, karma and rubocop to the default rake task

### PR DESCRIPTION
**Pull request by JiggyPete**
Ideally this will allow us to spot future issues long before they get to PR.

### CODE REVIEW

**Comment by neoeno** [comment]
ideal :+1:

**Comment by munckymagik** [comment]
:+1: for the things I didn't add. @neoeno @JiggyPete ?

File: Rakefile [hunk 1]
```diff
@@ -4,3 +4,27 @@
...
+
+KARMA_COMMAND = 'node_modules/.bin/karma'
+
+task :karma do
+  puts
+  puts 'Running JavaScript Karma specs'
+  puts
+
+  fail 'ERROR: karma is not installed' unless File.exist? KARMA_COMMAND
+  sh "#{KARMA_COMMAND} start spec/javascripts/karma.conf.js --single-run=true"
```
Comment (JiggyPete) [inline Rakefile]: Sweet @munckymagik!

**Comment by darokel** [comment]
:+1:

**Pull request merged**